src/compiler/frontend/expander.rs

```diff
@@ -98,22 +118,36 @@ impl Expander {
         operands: &[Datum],
         loc: Location,
     ) -> Result<Datum> {
-        let mut new_ls = vec![self.expand(operator)?];
+        let mut new_ls = vec![self.expand(operator)?.unwrap()];
         new_ls.extend(self.expand_all(operands)?);
 
         Ok(Datum::list(new_ls.into_iter(), loc))
     }
 
     fn expand_macro(&mut self, datum: &Datum, transformer: &syntax::Transformer) -> Result<Datum> {
         match transformer {
+            syntax::Transformer::LowLevel(expander) => {
```

Comment:
Both er- and lowlevel macros use the same mechanism to expand the form only with different arguments. Should this be consolidated? ER-macros might change more in the future when we track diverged bindings?